File: python programming/nested forloop7.py
import math
n=int(input('enter no of term:-'))
ans=0
for i in range(1,n+1):
 sum=0
 for j in range(1,i+1):
  print(j,end='')
  if j<i:
   print('+',end='')
  sum=sum+j
 print(f'/{i}!',end='')
 if i<n:
   print(' + ',end='')
 ans=ans+sum/math.factorial(i)
print("\n sum of seriese=",ans)
       

File: CppCompiler/nestedforloop7.cpp
#include<iostream>
using namespace std;

int main()
{
int i,j,n;
double k=1,h=1,sum=0,sum1=0;
cout<<"enter no of term";
cin>>n;
for(i=1;i<=n;i++)
{
if(i%2==0)
{
cout<<"1/";
k++;
for(j=1;j<=k;j++)
{
cout<<j;
if(j<k)
{
cout<<"+";
}
sum1=sum1+j;
}
sum=sum+1/sum1;
sum1=0;
}
else
{
cout<<"1/"<<++h;
sum=sum+1/h;
}
if(i<n)
{
cout<<" + ";
}
}
cout<<"sum of seriese="<<sum;
}

